proposals/threads/DataSegmentInitialization.md

```diff
@@ -0,0 +1,135 @@
+# Data segment initialization flag proposal
+
+This page describes a proposal for providing a flag to skip data segment
```

Comment:
Although it's not technically an issue yet b/c tables can't be shared, it would be good to keep elem and data segments symmetric and make the same change to both.